### PR TITLE
Begin work on swiftUI inbox

### PR DESCRIPTION
**Pull request by cobbal**
refs: MBL-14707
affects: none
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


[MBL-14707](https://instructure.atlassian.net/browse/MBL-14707)

#### Affected Apps: None
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/50165938/p/ccfb298ba19d195607a7bfabf16dadf6">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/50165942/p/00b5b9064357a8f290227db1b739aa2c">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/50165931/p/898755d92086ae73491852506c3c89f7">
    
  </a>
</details>

File: Core/Core/Extensions/UIImageViewExtensions.swift [hunk 1]
```diff
@@ -27,12 +28,7 @@ public struct LoadedImage {
 private var urlHandle: UInt8 = 0
 private var loaderHandle: UInt8 = 0
 
-public protocol ImageLoadingView: class {
-    var frame: CGRect { get }
-    func load(url: URL, didCompleteWith: LoadedImage?, error: Error?)
-}
-
-extension UIImageView: ImageLoadingView {
+extension UIImageView {
```
Comment (vanwagonet) [inline Core/Core/Extensions/UIImageViewExtensions.swift:31]: I've also been working on Avatar & loading remote images for swiftui, so we're going to conflict here.

**Review by vanwagonet — APPROVED** [review]
QA+1 Looks smooth. This will be nice when it's done.

**Pull request merged**